config/routes.rb

```diff
@@ -23,4 +23,6 @@
   # Chapter/11
   get "activate/:activation_digest", to: "account_activations#edit", as: "activate_user", constraints: { activation_digest: /.*/ }
 
+  # Chapter/12
+  resources :password_resets, except: %i(index show), constraints: { id: /.*/ }
```

Comment:
Mô tả về constraints

Reply:
Chấp nhận id là bất kì chuỗi nào, vì đôi khi sau params có dấu / / / nó sẽ hiểu là path khác và không match với route này dẫn đến việc route not found